### ISSUE TITLE
FTP synch

### ISSUE THREAD

**Issue by epaule** [WormBase/website#3941]
It would be very helpful if the main FTP site would synch from the EBI rather than Sanger, as some of the assembly hub files that are also used by JBrowse only exist on the EBI for policy reasons.

**Comment by epaule** [comment]
the URL is ftp://ftp.ebi.ac.uk/pub/databases/wormbase/releases/
and similar to the Sanger version, each release is a sub directory.

I added two softlinks to the dev and the live version, but you can ignore them

**Comment by tharris** [comment]
Done. Now mirroring from ebi. Looks like you have added some large new files? Can they be compressed?

**Comment by epaule** [comment]
they *are* compressed ;-)

The are HAL files from the progressive cactus. I extended it to all 29 genomes and there are another few subsampled ones in the lod/ directory, so the UCSC browser doesn't keel over when zooming out.

I also have a large collection of bigbed and wiggle files there, in case @scottcain wants some for JBrowse. I tested them on our JBrowse and parasite in addition to UCSC ... and as far as the respective browsers support them, they work very well :-)
